TileStache/Core.py

```diff
@@ -144,12 +144,21 @@
 """
 
 import logging
+from sys import modules
 from wsgiref.headers import Headers
-from StringIO import StringIO
-from urlparse import urljoin
+try:
+    from io import BytesIO
```

Comment:
What was the reason for using BytesIO here instead of StringIO? I ask this because in the next file (TileStache/Goodies/VecTiles/client.py), StringIO was preserved.